### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mjpaesold**
Please do it this time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
-# test
```
Comment (mjpaesold) [inline README.md]: Is this okay?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+# Test
 First test repository
```
Comment (mjpaesold) [inline README.md]: It did not have the . here...

File: README.md (hunk 1 above)
Comment (michaelpaesold) [inline README.md]: Yes!

File: README.md (hunk 2 above)
Comment (michaelpaesold) [inline README.md]: Yeah but you added a sentence!

**Comment by michaelpaesold** [comment]
Thank you!

**Pull request merged**